include/drawtypes/label.hpp

```diff
@@ -36,26 +37,31 @@ namespace drawtypes {
      * labels in a different way.
      */
     size_t m_maxlen{0_z};
+    // The default is set to RIGHT for backwards compatibility.
+    alignment m_alignment{alignment::RIGHT};
```

Comment:
We didn't have minlen before, why do we need to be backwards compatible here?

Am I forgetting something?